display_information/update-graphics/src/main/java/com/esri/samples/update_graphics/UpdateGraphicsSample.java

```diff
@@ -117,8 +116,8 @@ public void start(Stage stage) {
         }
       });
 
-      // add label, dropdown and buttons to the control panel
-      controlsVBox.getChildren().addAll(updateLocationButton, updateDescriptionButton, symbolLabel, symbolBox);
+      // add label, combo box and buttons to the control panel
```

Comment:
Another suggestion for plain english? Also, we only have one button now, by the looks of it? Would also be nice to order the items in the comment in the same order as they are in the code:

```suggestion
      // add the button, label, and combo box to the control panel
```